plugins/modules/system/lvol.py

```diff
@@ -477,9 +502,9 @@ def main():
                 size_requested = size_percent * this_vg['free'] / 100
 
             # Round down to the next lowest whole physical extent
-            size_requested -= (size_requested % this_vg['ext_size'])
+            size_requested -= (size_requested % this_vg['ext_size']) # I'm not sure why we are trying to calculate the size instead of letting the tool do it and pass it the VG|PVS|FREE options. In my opinion it will only lead to confusion since the module logic might not match the tools and descreptencies might arise (not actually taking all the free space but an extent less or some such scenario). We can leave the test/error logic, but the tool already does that as well. Besides you're not doing any checks for size for the -L option either, it keeps the code simple and I preffer that approach. Do you dissagree, any suggestions? At the very least we should use the same approach for both -l and -L for consistency. Will implement after a discussion. 
```

Comment:
A more readable version (increases the chance that someone actually reads it):

> I'm not sure why we are trying to calculate the size instead of letting the tool do it and pass it the VG|PVS|FREE options. In my opinion it will only lead to confusion since the module logic might not match the tools and descreptencies might arise (not actually taking all the free space but an extent less or some such scenario). We can leave the test/error logic, but the tool already does that as well. Besides you're not doing any checks for size for the -L option either, it keeps the code simple and I preffer that approach. Do you dissagree, any suggestions? At the very least we should use the same approach for both -l and -L for consistency. Will implement after a discussion.

(I'm not using this module and don't really know much about its internals, so I probably won't take part in this discussion.)

Reply:
No clue either - the only thing I know for sure is that this (long) comment is not helping at all.

Reply:
I added it as a comment since otherwise, I could not create the comment/review request on github on that line. 

Since there wasn't any input I tried implementing the feature with the minimum amount of changes made. See latest commits.